File: ClinicSystem/src/com/example/clinicsystem/controllers/C_Permission.java
package com.example.clinicsystem.controllers;


import com.example.clinicsystem.models.classes.M_Permission;

import java.util.List;

public class C_Permission {
    private M_Permission permission = new M_Permission();

    public List request(String requestType, String condition) {
        switch (requestType) {
            case "C":

            case "R":
                return permission.getAllPermissions(condition);
            case "U":

            case "D":

        }

        return null;
    }
}

File: clinic/C_Permission.php
<?php
include_once "M_Permission.php";

class C_Permission {


    public function request($requestType, $condition) {
        $permission = new M_Permission();

        switch ($requestType) {
            case "C":

            case "R":
                return $permission->getAllPermissions($condition);
            case "U":

            case "D":

        }

        return null;
    }
}
?>